secureboot_policy.go

```diff
@@ -293,13 +290,13 @@ func decodeSecureBootDb(r io.ReaderAt) ([]*efiSignatureData, error) {
 		// Decode EFI_SIGNATURE_DATA.SignatureOwner
 		signatureOwner, err := decodeEFIGUID(s)
 		if err != nil {
-			return false, fmt.Errorf("cannot decode SignatureOwner: %v", err)
+			return false, xerrors.Errorf("cannot decode SignatureOwner: %w", err)
 		}
 
-		// Obtain and decode the X509 certificate
+		// Obtain and signature data
```

Comment:
We can improve this comment later on another commit

Reply:
Ack